### ISSUE TITLE
Cannot encrypt self-trained MobileNet model.

### ISSUE THREAD

**Issue by whcjimmy** [IntelAI/he-transformer#20]
I try to train a MobileNet V2 model and want to run it in encrypted domain.

First, I used train_image_classifier.py in Tensorflow/models/research/slim to train a model.
The input is a 224 * 224 * 3 image.

Secondly, I used freeze_graph.py to get a pb file.

All related files can be downloaded below: https://drive.google.com/drive/folders/1e0Ix0oj_sAv4PzxBVyNoTVcMMU1pF5ZP?usp=sharing

However, the execution time of running model in encrypted domain are the same as the model running in plaintext domain.

In addition, even I set NGRAPH_HE_LOG_LEVEL=3, it didn't show any log such as encryption parameters.

So, I doubt that he-transformer doesn't run model in encryption domain correctly. 

Is there any way to fix this problem? 

Thanks!

**Comment by fboemer** [comment]
How are you running the trained model? I'm seeing `Attempting to use uninitialized value MobilenetV2/expanded_conv_1/project/BatchNorm/gamma` errors using the `test.py` script in our repo.

A few things to try:
* he-transformer uses TF v1.14; 
* The `frozen_model.pb` seems to have a FIFO_queue op, some `read` ops. Maybe cleaning up the `.pb` file by removing these
* Are you able to run inference using ngraph's CPU backend? You should see output such as
```2020-01-29 11:19:23.774580: I /ec/pdx/disks/aipg_lab_home_pool_02/fboemer/repos/he-transformer/build/ext_ngraph_tf/src/ext_ngraph_tf/ngraph_bridge/grappler/ngraph_optimizer.cc:239] NGraph using backend: CPU```

**Comment by whcjimmy** [comment]
Thanks for your help!

```read``` ops are really the problems!
